src/user/configure-metadata.md

```diff
@@ -1746,6 +1746,8 @@ Table: Indicator functions
 | least | (expression [, expression ...]) | Returns the least (lowest) value of the expressions given. Can be provided any number of arguments. |
 | log | (expression [, base ]) | Returns the natural logarithm (base e) of the numeric expression. If an integer is given as a second argument, returns the logarithm using that base. |
 | log10 | (expression) | Returns the common logarithm (base 10) of the numeric expression. |
+| null | | Returns no result. For example, if( #{FH8ab5Rog83}<0, null, 1 ) returns nothing if the data element value is less than 0, otherwise 1. |
```

Comment:
Maybe we could use _italic_ in the expression _if( #{FH8ab5Rog83}<0, null, 1 )_, just to emphasize the expression. Not a big deal though

Reply:
Thanks, good idea. Done!